### PR TITLE
document building without gettext, specifying openssl location

### PR DESCRIPTION
**Pull request by mykmelez**
These two additions to the README document fixes for issues I ran into while building the helper. They should help others get it built more quickly.

### CODE REVIEW

**Comment by aethanyc** [comment]
The instructions work for me when building the helper on macOS. Thank you for saving my time!

File: README.md [hunk 1]
```diff
@@ -56,6 +56,20 @@ have installed it using homebrew) you need to pass
 `PYTHON_PATH` environment variable to your Python installation path when using
 make to build this tool.
 
+If you don't have gettext, then you'll need to disable localization support
+in git when building the helper:
+
+  ```
+  $ NO_GETTEXT=1 make helper
```
Comment (glandium) [inline README.md]: make helper NO_GETTEXT=1 would be better.

File: README.md [hunk 2]
```diff
@@ -56,6 +56,20 @@ have installed it using homebrew) you need to pass
...
+  ```
+
+If you installed OpenSSL using homebrew on macOS, then you may need to specify
+the locations of its headers and libraries:
+
+  ```
+  $ make helper CFLAGS="-I/usr/local/opt/openssl/include" LDFLAGS="-L/usr/local/opt/openssl/lib"
```
Comment (glandium) [inline README.md]: It's probably simpler to suggest disabling openssl altogether: make helper NO_OPENSSL=1. The helper doesn't make real use of openssl, only for the sha1 hashing functions, and git has a replacement.

Come to think of it, make helper might as well just set both NO_GETTEXT=1 and NO_OPENSSL=1 by default.

**Comment by mykmelez** [comment]
> Come to think of it, make helper might as well just set both NO_GETTEXT=1 and NO_OPENSSL=1 by default.

Sure that seems reasonable, I've done it (and reverted the README changes) in 395d2bd.

**Comment by glandium** [comment]
Merged in 970abc2994ffe193d7a72005c582609f50643219

**Pull request closed without merging**